build.sbt

```diff
@@ -325,6 +325,9 @@ lazy val docs = project("docs")
   .settings(
     name := "akka-http-docs",
     resolvers += Resolver.jcenterRepo,
+    // In docs adding an unused variable can be helpful, for example
+    // to show its type
+    scalacOptions += "-Xlint:-unused",
```

Comment:
👍

Reply:
Had to look a few times to see the `-` to make sense of it :)